src/test/java/com/training/sample/tests/SampleGetJson.java

```diff
@@ -0,0 +1,24 @@
+package com.training.sample.tests;
+
+import com.training.sample.pages.SamplePage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class SampleGetJson {
```

Comment:
You can remove the Sample word, and just use GetJson - since this is a real unit test